wordle-web/src/views/GameView.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <div class="game">
+        <h1> This is the game page </h1>
+
+    <v-card 
+    class="mx-auto"
+    max-width="600"
+    max-height="400"
+    variant="outlined"
+    >
+      <v-card-title> Stirred Word </v-card-title>
+      <v-card-text>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Maecenas porttitor congue massa. Fusce posuere, magna sed pulvinar ultricies, purus lectus malesuada libero, sit amet commodo magna eros quis urna. Nunc viverra imperdiet enim. Fusce est. Vivamus a tellus. </v-card-text>
```

Comment:
```suggestion
      <v-card-text>
      Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Maecenas porttitor congue massa. Fusce posuere, magna sed pulvinar ultricies, purus lectus malesuada libero, sit amet commodo magna eros quis urna. Nunc viverra imperdiet enim. Fusce est. Vivamus a tellus. 
      </v-card-text>
```
Usually, the wrapping tags are on separate lines for large groups of text.